### PR TITLE
fix morph-tool versions

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

**Comment by asanin-epfl** [comment]
@GianlucaFicarelli or @matz-e can you please advice here? I don't see anything concrete in logs.

**Comment by asanin-epfl** [comment]
ok, It was py-six package. I wonder how is it related to my changes.

File: var/spack/repos/builtin/packages/py-bluepy/package.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class PyBluepy(PythonPackage):
     depends_on('py-pylru@1.2:', type='run', when='@:2.2.9')
 
     # morph-tool
-    depends_on('py-morph-tool@2.4.3:2.999', type='run', when='@2.3.0:')
+    depends_on('py-morph-tool@2.5.1:2.999', type='run', when='@2.3.0:')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-bluepy/package.py:43]: Why this but no version bump?

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ class PyBluepyemodel(PythonPackage):
     depends_on('py-h5py@2.9:', type='run')
     depends_on('py-ipyparallel@6.3:', type='run')
     depends_on('py-matplotlib@2.2:', type='run')
-    depends_on('py-morph-tool@2.4.4:', type='run')
+    depends_on('py-morph-tool@2.5.1:', type='run')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-bluepyemodel/package.py:31]: Same?

File: var/spack/repos/builtin/packages/py-bluepy/package.py (hunk 1 above)
Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-bluepy/package.py:43]: I've deleted morph-tool releases 2.4.2, 2.4.3, 2.4.4 because they were incorrect.

**Review by matz-e — DISMISSED** [review]

**Review by GianlucaFicarelli — DISMISSED** [review]

**Comment by asanin-epfl** [comment]
retest this please

**Comment by asanin-epfl** [comment]
retest this please

**Comment by asanin-epfl** [comment]
@matz-e what should I do with errors like this:
```
[2021-06-08T15:30:22.958Z]  > git fetch --tags --progress ssh://bbpcode.epfl.ch/hpc/spack-licenses +refs/heads/*:refs/remotes/origin/*

[2021-06-08T15:30:23.756Z] ERROR: Error fetching remote repo 'origin'

[2021-06-08T15:30:23.756Z] hudson.plugins.git.GitException: Failed to fetch from ssh://bbpcode.epfl.ch/hpc/spack-licenses
```
?

**Comment by asanin-epfl** [comment]
retest this please

**Comment by asanin-epfl** [comment]
retest this please

File: var/spack/repos/builtin/packages/py-morphio/package.py [hunk 3]
```diff
@@ -17,6 +17,7 @@ class PyMorphio(PythonPackage):
     version('unifurcation', branch='unifurcation', submodules=True, get_full_repo=True)
 
     version('3.1.1', sha256="ad9f0e363f09f03c6eda54f5f3b006d204236677d2f2c9675421e0441033a503")
+    version('2.7.1', sha256="3f3e2229da85e874527775fce080f712b6dc287edc44b90b6de35d17b34badff")
```
Comment (GianlucaFicarelli) [inline var/spack/repos/builtin/packages/py-morphio/package.py:20]: Shouldn't we update also the spack package for `morphio` when `py-morphio` is updated?

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/py-morphio/package.py:20]: thank you! I didn't know that the package is duplicated. Why there are `morphio` and `py-morphio`?

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-morphio/package.py:20]: Because it's hard to build both at the same time :shrug:

**Review by GianlucaFicarelli — APPROVED** [review]

**Review by matz-e — APPROVED** [review]

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**